### PR TITLE
Add support for alternative CSV format

### PR DESCRIPTION
**Pull request by markus456**
Adds rudimentary CSV parsing of payment records in the form of the parseCSV function. Also added unit tests with the currently known format of the CSV schema. Changes to the format are somewhat easy to do since it uses DictReader.

Added a new button to the transactions admin page for uploading the new CSV files. The old button for the NDA forms is left there for backwards compatibility.

Fixed the parser.py unit test to use the python from the environment instead of a hard-coded path. This makes it work with venv.

### CODE REVIEW

**Comment by markus456** [comment]
@rambo if possible, please review this change. I suspect that my Django usage is not quite appropriate.

**Review by rambo — APPROVED** [review]
it's been ages since I touched Django but it doesn't look any worse than the original to current me(These days I'd expect any new code to pass `mypy --strict`)

File: project/ndaparser/parser.py [hunk 1]
```diff
@@ -126,6 +127,22 @@ def ascii2scandic(string):
...
 
 
+def parseCsv(f):
+    transactions = []
+    for t in DictReader(f, delimiter=';'):
+        if not t["Viitenumero"]:
+            continue
+        amount = int(float(t["Määrä"].replace(",", ".")))
+        timestamp = datetime.strptime(t["Kirjauspäivä"], "%Y/%m/%d").date()
+        archive_id = "TODO: We need the archive ID"
```
Comment (rambo) [inline project/ndaparser/parser.py]: Change this to something that makes for example sha1 from the information of the transaction. Actual Nordea transaction-ids will not be available.

**Review by rambo — CHANGES_REQUESTED** [review]

File: project/ndaparser/parser.py (hunk 1 above)
Comment (markus456) [reply project/ndaparser/parser.py]: I did as you suggested and went with the SHA1 of the CSV record fields concatenated into a single string.

**Comment by jssmk** [comment]
Good job updating the test file, starts to look good to me

**Pull request merged**